test/suites/metrics.sh

```diff
@@ -11,6 +11,10 @@ test_metrics() {
   lxc project create foo -c features.images=false -c features.profiles=false
   lxc launch testimage c3 --project foo
 
+  # create but dont start a container in separate non default project to check for stopped instance accounting.
+  lxc project create foo2 -c features.images=false -c features.profiles=false
+  lxc init testimage c4 --project foo2
```

Comment:
While just `init`ing a testimage based container is quite cheap, for instance that we never intend to start, we could probably get faster runs with `--empty`. I'm happy to tackle that in future PR unless you think it's not worth it or is too unrealistic?

Reply:
thanks!